tests/Statable/StatableArticle.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Sebdesign\SM\Test\Statable;
+
+use Sebdesign\SM\Traits\Statable;
+
+class StatableArticle
+{
+    use Statable;
+
+    const HISTORY_MODEL = [
```

Comment:
If we use a polymorphic model in the package, we don't need the history model constant.
Maybe a property or a function so the developer can override/extend the history model.